WEB_SERVICE/WEB/_POTENTIALLY/SSL_EXTENSION_BLEED/description.md

```diff
@@ -0,0 +1,15 @@
+SSL Extension Bleed is a vulnerability that exploits improper handling of SSL/TLS extensions during the handshake process. It can lead to the exposure of sensitive information, such as plaintext data from previous connections, session keys, or even private keys.
+
+This vulnerability allows attackers to:
+
+- **Extract Sensitive Information:** Attackers can access data stored in memory, potentially revealing SSL session keys or plaintext data transmitted over earlier SSL connections.
+
+- **Hijack Sessions:** If session keys are disclosed, attackers may gain unauthorized access to existing SSL sessions, allowing them to access user accounts or sensitive information without permission.
+
+**Example Scenario:**
+
+Imagine a web server that supports SSL/TLS and allows a specific extension, such as `SessionTicket TLS`, to optimize session resumption during the handshake process. However, due to improper memory management or a flaw in the handling of these extensions, some sensitive data from previous SSL sessions (such as session keys or portions of plaintext data) remains in memory.
+
+An attacker who can repeatedly initiate SSL/TLS handshakes may craft malicious SSL extensions or probe the server in a way that causes the server to unintentionally leak data from memory during the handshake. As a result, the attacker might recover session keys used in earlier connections, which could then be used to decrypt previously encrypted communication or hijack active user sessions.
```

Comment:
I would  expect more details on how these malicious SSL extensions or how the server can unintentionally leak data..